cassandra ttl_test.py::TestDistributedTTL::test_ttl_is_replicated true true
log_dts_dir = /users/masix/vm_images/ZebraConf/app_meta/logs
the_test is ttl_test.py::TestDistributedTTL::test_ttl_is_replicated
rm: cannot remove '/tmp/my_log.txt': No such file or directory
============================= test session starts ==============================
platform linux -- Python 3.6.9, pytest-3.6.4, py-1.11.0, pluggy-0.7.1 -- /users/masix/dtest/bin/python
cachedir: .pytest_cache
rootdir: /users/masix/cassandra-dtest, inifile: pytest.ini
plugins: flaky-3.7.0, timeout-1.4.2, repeat-0.9.1
timeout: 900.0s
timeout method: signal
timeout func_only: False
collecting ... collected 1 item

ttl_test.py::TestDistributedTTL::test_ttl_is_replicated 
-------------------------------- live log setup --------------------------------
22:50:55,55 ccmlib.node WARNING [msx] ccmlib:node.py:start(), self.name= node1, self.pid= 2297
22:50:55,57 ccmlib.cluster WARNING [msx] ccmlib:cluster.py:start() p.pid= 2200 node.name= node1 node.pid= 2297
22:51:03,24 ccmlib.node WARNING [msx] ccmlib:node.py:start(), self.name= node2, self.pid= 2762
22:51:03,25 ccmlib.cluster WARNING [msx] ccmlib:cluster.py:start() p.pid= 2665 node.name= node2 node.pid= 2762
PASSED                                                                   [100%]
===Flaky Test Report===

test_ttl_is_replicated passed 1 out of the required 1 times. Success!

===End Flaky Test Report===

========================== 1 passed in 32.41 seconds ===========================
[msx] rc = 0
